.azure-pipelines/azure-pipelines.yml

```diff
@@ -178,6 +180,8 @@ stages:
               test: freebsd/11.4
             - name: FreeBSD 12.2
               test: freebsd/12.2
+            - name: FreeBSD 13.0
+              test: freebsd/13.0
```

Comment:
Do we want to remove FreeBSD 11 in `devel`? It was [end of life on September 30, 2020](https://www.freebsd.org/security/unsupported/).

Reply:
☝️ @mattclay

Reply:
I think 11.4 is still supported, it's listed on the main page: https://www.freebsd.org
And [this](https://www.freebsd.org/security/#sup) says it's supported until September of this year.

Reply:
Ah, gotcha. So I suppose we just need to decide if we want to expand the matrix or trim 11.4.

Reply:
Since FreeBSD 11.x is expected to be EOL before Ansible 2.12 is released, we should remove it from CI now and replace it with FreeBSD 13.0.

Once this is merged we should notify collection maintainers so they can make the same changes to their CI configurations. Then as we get closer to releasing Ansible 2.12 (or when collections have all been updated) we can remove support for FreeBSD 11.x from ansible-test.